### DISCUSSION TITLE
QQ适配器在QQ频道私聊情况下机器人不收发

### DISCUSSION THREAD

**Discussion by SiJin233** [nonebot/discussions#423] [问答]
如题，在使用QQ适配器登录QQ频道机器人后，私聊机器人日志不收也不发，而群聊一切正常

**Comment by yanyongyu** [comment]
非上线机器人无法接收私聊，请阅读QQ官方文档

**Reply by SiJin233** [reply]
呃，Q频机器人已经审核并私域上线啦，然后我在查阅文档后并没有发现能解决问题的信息..请问能再给一些提示和帮助嘛？😢

**Reply by yanyongyu** [reply]
你是否订阅了 `DIRECT_MESSAGE` intent
